deploy/microshift-bootc/README.md

```
@@ -0,0 +1,420 @@
+```bash
+make bootc-run
+```
+
+This will:
+- Create a 1GB LVM disk image at `/var/lib/microshift-okd/lvmdisk.image`
+- Start MicroShift in a privileged container
+- Set up LVM volume groups inside the container for TopoLVM
+- Wait for MicroShift to be ready
+
+**Output example:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)